bin/isolated-functions.sh

```diff
@@ -460,6 +460,28 @@ if [[ -z ${XARGS} ]] ; then
 	esac
 fi
 
+makeopts_jobs() {
+	# Copied from eutils.eclass:makeopts_jobs()
+	local jobs=$(echo " ${MAKEOPTS} " | \
+		sed -r -n 's:.*[[:space:]](-j|--jobs[=[:space:]])[[:space:]]*([0-9]+).*:\2:p')
+	echo ${jobs:-1}
+}
+
+# Run ${XARGS} in parallel for detected number of CPUs, if supported.
+# Passes all arguments to xargs, and returns its exit code
+_parallel_xargs() {
```

Comment:
Let's used a triple underscore name prefix for the new functions, for automatic filtering with `unset -f $(compgen -A function ___)` in __save_ebuild_env.